user-manuals/data-manager/configurations.md

```diff
@@ -65,6 +65,36 @@ Editing an existing Data Manager Configuration allows users to update configurat
 Config ID cannot be changed after creation. Create a new Config if a new Config ID is needed.
 {% endhint %}
 
+## Verify the Data flow of Data Configurations
+
+In HotWax Commerce, the significance of the Data Manager Configurations feature extends beyond the management of data flow. A crucial aspect of this tool lies in the Data Manager Logs, which play a pivotal role in ensuring the accuracy and completeness of data imported into the Order Management System (OMS). These logs provide essential insights into the status of imported data, allowing users to verify the success of their operations and identify potential issues promptly.
+
+### Steps
+
+
+1. Search for the Desired Service:
+
+- Navigate to the search bar within HotWax Commerce Data Manager Confirgurations page.
+- Search service with the name of the desired service.
+
+2. Open the Import Data for the Service:
+
+- Identify the relevant service in the search results.
+- Click on the blue open icon located in front of the service name.
+- The system will open the [Import Data page](data-manager/imports.md) for the selected service.
+
+3. Check the Status of the Service:
+
+- Review the displayed information to check the status of the service.
+- Ensure that the status of the service is finished. You can view different data status types [here.](https://docs.hotwax.co/user-guides/workflow/data-manager/imports#data-manager-logs)
+
+**View Logs for Detailed Information:**
```

Comment:
We need to add the point where one of the data failed, so we can read the failed record, and understand the issue in the failed record.